pkg/storage/calico/bgpConfiguration_storage.go

```diff
@@ -56,8 +56,8 @@ func NewBGPConfigurationStorage(opts Options) (registry.DryRunnableStorage, fact
 		versioner:         etcd.APIObjectVersioner{},
 		aapiType:          reflect.TypeOf(aapi.BGPConfiguration{}),
 		aapiListType:      reflect.TypeOf(aapi.BGPConfigurationList{}),
-		libCalicoType:     reflect.TypeOf(libcalicoapi.BGPConfiguration{}),
-		libCalicoListType: reflect.TypeOf(libcalicoapi.BGPConfigurationList{}),
+		libCalicoType:     reflect.TypeOf(api.BGPConfiguration{}),
```

Comment:
libCalicoType / libCalicoListType are misnamed now. Actually, I suspect they can probably be removed entirely.

along with all of the conversion functions, for example: https://github.com/projectcalico/apiserver/blob/93a95ac6df954a7c648f03143108de69bc299e5f/pkg/storage/calico/bgpConfiguration_storage.go#L77-L112